design-docs/CLI-Subscriptions.md

```diff
@@ -237,11 +237,11 @@ Options:
 
 ```
 planet subscriptions request \
-    --source $(planet subscriptions request-catalog
-        --item-type PSScene
-        --asset-types ortho_analytic_8b_sr,ortho_udm2
-        --geom aoi.json
-        --start-time 05/01/2022)
+    --source $(planet subscriptions request-catalog \
```

Comment:
- The `request` call is missing its `--name` flag.
- I really like this example, but I would recommend splitting this example into 2 parts:
1. A direct call to `request-catalog`, which you wrote in your PR:
```
planet subscriptions request-catalog \
        --item-types PSScene \
        --asset-types ortho_analytic_8b_sr,ortho_udm2 \
        --geometry aoi.geojson \
        --start-time 2022-01-01
```
Maybe even have this write to `source.json`?

2. Either the `request` example you wrote here, showing how you can call the function inline OR a `request` command with `source.json` as an output from the example 1. For instance, either
```planet subscriptions request \
    --name "some name" \
    --source $(planet subscriptions request-catalog \
        --item-types PSScene \
        --asset-types ortho_analytic_8b_sr,ortho_udm2 \
        --geometry aoi.geojson \
        --start-time 2022-01-01) \
    --delivery delivery.json | planet subscriptions create -
```
OR
```
planet subscriptions request \
    --name "some name" \
    --source source.json \
    --delivery delivery.json | planet subscriptions create -
```